django_irods/views.py

```diff
@@ -172,32 +164,12 @@ def download(request, path, rest_call=False, use_async=True, use_reverse_proxy=T
     # if none of these are set, it's a normal download
 
     # determine active session
-    if res.is_federated:
-        # the resource is stored in federated zone
+    if icommands.ACTIVE_SESSION:
+        if __debug__:
+            logger.debug("using ACTIVE_SESSION")
         session = icommands.ACTIVE_SESSION
     else:
-        # TODO: From Alva: I do not understand the use case for changing the environment.
```

Comment:
Did you delete this based on the TODO?  Are we not using these settings?

Reply:
@sblack-usu I was not deleting this based on the TODO, but actually investigated the code and made sure we are not using these settings. I am pretty certain we are not using them.